engines/sqlite.py

```diff
@@ -10,13 +10,13 @@ class engine(Engine):
     name = "SQLite"
     abbreviation = "sqlite"
     datatypes = {
-        "auto": "INTEGER",
+        "auto": ("INTEGER", "AUTOINCREMENT"),
         "int": "INTEGER",
         "bigint": "INTEGER",
         "double": "REAL",
         "decimal": "REAL",
         "char": "TEXT",
-        "bool": "INTEGER",
+        "bool": "REAL",
```

Comment:
Seems like boolean should stays as "INTEGER" in SQLite:
https://www.sqlite.org/datatype3.html#section_2_1